src/parsing.jl

```diff
@@ -162,26 +162,70 @@ function parse_row!(rec::Records, row::Int, bytes, pos, len, options, eol_option
     return pos, code
 end
 
-function parse_row!(rec::TwoWindingTransformers, row::Int, bytes, pos, len, options, eol_options)
-    # Each `TwoWindingTransformers` is 4 lines of the file
-    # cols_per_line = (14, 3, 16, 2)
-    eol_cols = (14, 17, 33, 35)
-    ncols = nfields(rec)
-    @assert ncols == last(eol_cols)
+
+###
+### transformers
+###
+
+# Transformers data is a bit special, as records have 2 possible schemas
+# see https://github.com/nickrobinson251/PowerFlowData.jl/issues/17
+#
+# Each "two-winding transformer" is 4 lines with (14, 3, 16, 2) columns each, and
+# each "three-winding transformer" is 5 lines with (14, 11, 16, 16, 16) columns each.
+const T2_COLS = (14,  3, 16,  2)
+const T3_COLS = (14, 11, 16, 16, 16)
+const EOL_COLS = cumsum(T3_COLS)
```

Comment:
i'm not sure these consts are better than magic numbers (tbh i find them harder to think about), but i suspect they might survive the test of time better...